.github/workflows/bridge-ui-preview.yml

```diff
@@ -13,6 +13,7 @@ on:
 jobs:
   Deploy-Preview:
     runs-on: [taiko-runner]
+    if: github.actor != 'dependabot[bot]'
```

Comment:
Dave, won't incompatible dependency upgrades break the build?

Reply:
i didn't know i can have secrets for dependabot in github settings. i added them now, so this job should not fail anymore. i agree it's nice to build a preview as part of an "e2e test" after dependency upgrades.